### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by berna-ylmz** [IDKiro/action-recognition-pytorch#10]
Hi,
I trained using LSTM-CNN with my own dataset,but results are not good.Loss parameter is not decreasing.
What do you suggest ? Is"batch normalization" a good idea? If yes , Where would it be appropriate to put?
Thanks .

**Comment by IDKiro** [comment]
The backbone network was trained with ImageNet dataset.
Make sure that you do the right normalization:
```
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                 std=[0.229, 0.224, 0.225])
```
BN is a common tool to stablize the training, but it is harmful when using small batch size.
Actually, BN has been used in the trained backbone network (the parameters of the backbone network won't be updated when training).
I do not focus on action recognition, so I would recommend you to ask other ones.

**Comment by berna-ylmz** [comment]
Ok,thanks
